scania-digital-design-system/sdds
# Feature - add travis eslint

**Describe pull-request**  
_Describe what the pull-request is about_

Enabling first rules of eslint to get going

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #

**How to test**  
_Add description how to test if possible_
1. Travis should pass and run npm run lint command
2. Check storybook

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
